.ci/push_docker.sh

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+# This script is present on workers but may not be present in a development
+# environment.
+
+if [ ${WORKSPACE+x} ]  # We are on a CI worker
+then
+  source /usr/local/bin/bash_standard_lib.sh
+fi
+
+readonly RETRIES=3
+
+readonly DOCKER_REGISTRY_URL="$1"
+
+readonly DOCKER_IMAGE_NAME="$2"
+
+readonly DOCKER_TAG="$3"
+
+readonly DOCKER_PUSH_IMAGE="$DOCKER_REGISTRY_URL/$DOCKER_IMAGE_NAME:$DOCKER_TAG"
+
+readonly DOCKER_PUSH_IMAGE_LATEST="$DOCKER_REGISTRY_URL/$DOCKER_IMAGE_NAME:latest"
+
+echo "INFO: Pushing image $DOCKER_PUSH_IMAGE to $DOCKER_REGISTRY_URL"
+
+if [ ${WORKERS+x} ]  # We are on a CI worker
```

Comment:
I'm not much familiar with `WORKERS` but `CI` is the default env variable if it helps

Reply:
I copied this logic from the [Java agent's push_docker.sh](https://github.com/elastic/apm-agent-java/blob/main/scripts/jenkins/push_docker.sh)